### DISCUSSION TITLE
Spartan Support

### DISCUSSION THREAD

**Discussion by SonnenPingu** [Zrips/CMI#7999] [Feature Requests]
Hello.

It can't be that two paid plugins, both used by x people, are not compatible with each other. 

Spartan developer, can no longer provide support for cmi. He has already done all he can. 

So it is up to the developer of cmi to establish the support. 

There is an api interface from Spartan, so it is quite easy for developers to make it compatible. 

https://docs.google.com/document/d/e/2PACX-1vTzeCDiD2goCFO3oWIs3ANwmOvOqI2TLYVkjHO4Z-2q3yTDZPyYZ9bFX4OCwlBamWA_SOP7Ib5uARc8/pubub

**Comment by CloudeLecaw** [comment]
well you might should descripte what went wrong?

Else i dont think Zrips can take any actions here.
might be a thing for issues?

**Reply by SonnenPingu** [reply]
It's a feature request. No issue. 

BUT sure, I could also open an issue. 

As soon as a bedrock player uses CMI warps, he is marked with inventory click by spartan. Thus, cmi and geyser and spartan are not fully compatible with each other. Spartan can't do anything anymore, so Zrips has to do it.

**Comment by CloudeLecaw** [comment]
CMI has no bedrock support at the moment for bedrock players

only paper/spigot/(and a bit folia)  - with java client editions

all other server/client types have a 50 / 50 it can work if not then not.
but at least its a request for a nur supported jet server type so why not ;)

Hands up for it xD

**Reply by SonnenPingu** [reply]
It shouldn't be difficult for Zrips to support GeyserMC. Even a free plugin can do that. In fact, a paid plugin is expected to have bedrock support. Lol

Well, the guis and co work. The funny thing is that spartan then has a corresponding false detection. Because the combination: GeyserMC, CMI and Spartan are not fully compatible. 

In a way, cmi works with bedrock players. In fact, it's pretty good 

I have a PaperMc server. The bedrock support is provided by GeyserMC.

**Comment by CloudeLecaw** [comment]
many chat things are not supported in bedrock because players cant click it ;)
bedrock has no events for it. but yeah i aspect many things do work on bedrock as they should
